tests/dummy/app/routes/nodes/detail/children.js

```diff
@@ -2,10 +2,7 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
     model() {
-        return this.store.query('node', {
-            filter: {
-                parent: this.modelFor('nodes.detail').get('id')
-            }
-        });
-    }
+        let node = this.modelFor('nodes.detail');
+        return node.get('children');
```

Comment:
Can you explain the reason for this change?

By default ember doesn't support paginating over relationships, which would limit this query as written to the first 10 results. If you need more, I can help point you to a few solutions available in our codebase.